internal/project/golang/toolchain.go

```diff
@@ -127,7 +127,7 @@ func (toolchain *Toolchain) CompileDockerfile(output *dockerfile.Output) error {
 		From("${TOOLCHAIN}")
 
 	if toolchain.Kind == ToolchainOfficial {
-		packages := []string{"add", "bash", "curl", "build-base", "protoc", "protobuf-dev"}
```

Comment:
we have toolchain.ExtraPackages for that, no need to push to the base image

Reply:
https://github.com/talos-systems/conform/blob/master/.kres.yaml#L16-L20

Reply:
Doh!